packages/gdl-frontend/components/GlobalMenu/CircleLabel.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+/**
+ * Part of GDL gdl-frontend.
+ * Copyright (C) 2018 GDL
+ *
+ * See LICENSE
+ */
+import React from 'react';
+import { FiberManualRecord as CircleLabel } from '@material-ui/icons';
+import type { ReadingLevel } from '../../types';
+
+const colorMap = {
```

Comment:
The same color map is used 3 places. I think this can be refactorized, maybe under the style directory.